scripts/compile.sh

```diff
@@ -45,6 +49,19 @@ for arg in "$@"; do
     esac
 done
 
+if [ ! -z "$REL_BUILD" ]; then
+    BUILD_TYPE=Release
+elif [ ! -z "$DEV_BUILD" ]; then
+    BUILD_TYPE=Debug
+else
+    BUILD_TYPE=RelWithDebInfo
```

Comment:
So this is the default now?

Reply:
Right, the default has not been changed.